tools/ctl/ctl.go

```diff
@@ -229,34 +271,181 @@ func makeHTTPRequest(ctx context.Context, u *url.URL, msg schema.Message) *http.
 	return req
 }
 
+type WorkerConfig struct {
+	client   *http.Client
+	url      *url.URL
+	limiters map[string]<-chan time.Time
+	run      string
+}
+
+type AttestWorker struct {
+	WorkerConfig
+}
+
+func (w *AttestWorker) processOne(ctx context.Context, p benchmark.Package, out chan schema.Verdict) {
+	for _, v := range p.Versions {
+		<-w.limiters[p.Ecosystem]
+		resp, err := w.client.Do(makeHTTPRequest(ctx, w.url.JoinPath("rebuild"), schema.RebuildPackageRequest{
+			Ecosystem: rebuild.Ecosystem(p.Ecosystem),
+			Package:   p.Name,
+			Version:   v,
+			ID:        w.run,
+		}))
+		var errMsg string
+		if err != nil {
+			errMsg = errors.Wrap(err, "sending request").Error()
+		}
+		if resp.StatusCode != 200 {
+			errMsg = errors.Wrapf(errors.New(resp.Status), "sending request").Error()
+		}
+		var verdict schema.Verdict
+		if errMsg != "" {
+			verdict = schema.Verdict{
```

Comment:
wdyt about assining the Target above and conditionally set verdict.Message? Would cut down on the vertical space here